src/NServiceBus.RabbitMQ/Administration/QueueCreator.cs

```diff
@@ -6,13 +6,13 @@
     class QueueCreator : ICreateQueues
     {
         readonly ConnectionFactory connectionFactory;
-        readonly IRoutingTopology topology;
+        readonly IRoutingTopology routingTopology;
```

Comment:
Minor point, but I would have preferred to have this rename (and the similar one further down) as a separate commit to reduce the noise around the actual change. No need to amend though.